modules/account-tokenizer/src/lib.rs

```diff
@@ -0,0 +1,385 @@
+		AccountAlreadyRequestedMinted,
+		/// The mint request for the given account is not found.
+		MintRequestNotFound,
+		/// The confirmed owner and the mint requester isn't the same.
+		MintRequestDifferentFromOwner,
+		/// The account's NFT token cannot be found.
+		AccountTokenNotFound,
+		/// The caller is unauthorized to make this transaction.
+		CallerUnauthorized,
+	}
+
+	#[pallet::event]
+	#[pallet::generate_deposit(fn deposit_event)]
+	pub enum Event<T: Config> {
+		/// The class ID of Account Tokens has been set
+		NFTClassIdSet {
+			class_id: ClassIdOf<T>,
+		},
+		/// The mint fee has been set
+		MintFeeSet {
+			mint_fee: Balance,
+		},
+		/// The burn fee has been set
+		BurnFeeSet {
+			burn_fee: Balance,
+		},
+		/// The deposit amount for request_mint has been set
+		RequestMintDepositSet {
+			deposit: Balance,
+		},
+		/// The user has requested to mint a Account Token NFT.
+		MintRequested {
+			account: T::AccountId,
+			who: T::AccountId,
+		},
+		MintRequestRejected {
+			account: T::AccountId,
+			who: T::AccountId,
+		},
+		/// A NFT is minted to the owner of an account on the Relaychain.
+		AccountTokenMinted {
+			account: T::AccountId,
+			owner: T::AccountId,
+			token_id: T::TokenId,
+		},
+		/// The account token is burned, the control of the `account` on the relaychain is
+		/// relinquished to `new_owner`.
+		AccountTokenBurned {
+			account: T::AccountId,
+			owner: T::AccountId,
+			token_id: T::TokenId,
+			new_owner: T::AccountId,
+		},
+	}
+
+	/// Stores the NFT's class ID. Settable by authorized Oracle. Used to mint and burn PRTs.
+	#[pallet::storage]
+	#[pallet::getter(fn nft_class_id)]
+	type NFTClassId<T: Config> = StorageValue<_, ClassIdOf<T>, OptionQuery>;
+
+	/// Stores accounts that are already minted as an NFT.
+	/// Storage Map: Tokenized Account Id  => NFT Token ID
+	#[pallet::storage]
+	#[pallet::getter(fn minted_account)]
+	type MintedAccount<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, T::TokenId, OptionQuery>;
+
+	/// Stores mint requests.
+	/// Storage Map: Account to be tokenized  => Requester's Account Id
+	#[pallet::storage]
+	#[pallet::getter(fn mint_requests)]
+	type MintRequests<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, T::AccountId, OptionQuery>;
+
+	/// The amount of fee paid when minting a Account Token
+	#[pallet::storage]
+	#[pallet::getter(fn mint_fee)]
+	type MintFee<T: Config> = StorageValue<_, Balance, ValueQuery>;
+
+	/// The amount of fee paid when burning a Account Token, in additional to the XCM cost.
+	#[pallet::storage]
+	#[pallet::getter(fn burn_fee)]
+	type BurnFee<T: Config> = StorageValue<_, Balance, ValueQuery>;
+
+	/// Deposit locked when requesting to mint. If the mint request is successful, it is returned.
+	/// Otherwise the deposit is confiscated as penalty.
+	#[pallet::storage]
+	#[pallet::getter(fn request_mint_deposit)]
+	type RequestMintDeposit<T: Config> = StorageValue<_, Balance, ValueQuery>;
```

Comment:
```suggestion
	type MintRequestDeposit<T: Config> = StorageValue<_, Balance, ValueQuery>;
```

This should just be a constant.